contents/blog/best-open-source-analytics-tools.md

```diff
@@ -11,221 +11,301 @@ featuredImageType: full
+
+This section is dedicated to tools that actively collect data on user behavior. These range from the very simple, such as the privacy-focused Plausible, to broad analytics tools like Matomo.
+
+### PostHog
 ![PostHog - best open source analytics tools](../images/blog/gdpr-compliant-analytics/posthog-gdpr-compliant.png)
 
-PostHog is an all-in-one open-source analytics platform that offers all the essential tools you need to build better products, including funnels, feature flags, path analysis, A/B testing, heat maps, and more – it's also one of the [best open source session recording tools](/blog/best-open-source-session-replay-tools) around. 
+- **GitHub Stars:** 8.9k
+- **Language(s):** TypeScript (53%) and Python (42%)
+- **Alternative to:** Mixpanel, Amplitude, Hotjar, FullStory, LaunchDarkly
+- [PostHog on GitHub](https://github.com/PostHog/posthog)
 
-It’s entirely self-serve, with the ability to auto-capture event information out of the box. The interface makes querying your data a breeze, with no SQL required. 
+[PostHog](https://posthog.com/) is an all-in-one analytics platform designed to give engineers, product managers and data scientists a complete view of user behavior. It's built on the highly-scalable ClickHouse OLAP database.
 
-Best of all, PostHog can be self-hosted on your existing infrastructure — making it an excellent alternative to tools such as Mixpanel if you’re concerned about privacy. 
+Features include a comprehensive product analytics suite (e.g. trend analysis, dashboards, funnels, path analysis, retention tracking etc.), feature flags for feature control, experimentation (A/B and multivariate testing), session recording, heatmaps and [more](/product).
 
-Of course, we _would_ say PostHog is great, but if you need convincing then you can [get started for free](/pricing) and decide for yourself. You can self-host, but [PostHog Cloud is free up to 1 million events per month](/pricing#cloud) if you'd prefer a fully-managed experience.
+It’s entirely self-serve, with the ability to auto-capture event information out of the box. PostHog can also be self-hosted on your existing infrastructure, making it ideal if you don't want to share data with third parties. It's a good option if you're after an [open-source session recording tool](/blog/best-open-source-session-replay-tools), too. 
 
-### Who is PostHog for?
-PostHog is ideal for engineering, data science, and product teams. As an all-in-one platform, it combines the functionality of traditional product analytics platforms with tools like Feature Flags, Session Recording and A/B testing, making it ideal for teams who want a complete view of their product without the hassle of running multiple separate tools.
+#### Who is PostHog for?
+PostHog is useful for engineering, data science, and product teams. As an all-in-one platform, it can replace multiple tools (e.g. Mixpanel, LaunchDarkly, Hotjar) so you can simplify your tech stack. 
```

Comment:
```suggestion
PostHog is useful for engineering, data science, and product teams. As an all-in-one platform, it can replace multiple tools (e.g. Mixpanel, LaunchDarkly, Hotjar) so you can simplify your tech stack. [PostHog customers](/customers) include the likes of [Hasura](/customers/hasura), [Netdata](/customers/netdata), and [Phantom](/customers/phantom).  
```

Reply:
@joethreepwood Slightly different approach. Less promotional but still effective, I think. I chose the two largest open source companies we have case studies for. Might switch our Phantom for YC when that goes live.